web/modules/custom/joinup_subscription/joinup_subscription.behat.inc

```diff
@@ -70,30 +72,61 @@ class SubscriptionSubContext extends DrupalSubContextBase implements DrupalSubCo
   }
 
   /**
-   * Subscribes the given users to the given entity.
+   * Subscribes the given users to the given discussion.
    *
    * @param \Behat\Gherkin\Node\TableNode $table
    *   A table with the keys 'username' and 'title'.
-   * @param string $bundle
-   *   The entity bundle to which the entity with given title corresponds.
-   * @param string $entity_type
-   *   The entity type to which the entity with given title corresponds.
    *
-   * @Given (the following ):bundle :entity_type subscriptions:
+   * @Given (the following )discussion subscriptions:
```

Comment:
This step definition was conflicting with the step definition below so I reworked it to be unique. We have no plans to support anything other than discussions right now.